utils/parameters.py

```diff
@@ -14,8 +14,4 @@ def __init__(self):
         self._parameters_file = yaml.load(stream)
 
     def get_parameter(self, parameter):
-        for key, value in self._parameters_file.iteritems():
-            if key == parameter:
-                return value
-
-        raise Exception('Key %s not found' % parameter)
\ No newline at end of file
+        return self._parameters_file[parameter]
```

Comment:
And what about the exception handling? 🤔

Reply:
You weren't handling it, just raising it. In this case it will raise too whenever the key doesn't exist in the dict.

Reply:
👺

Reply:
Okey. Merging. In future projects I'll need to come back to this class so better to have it updated 🐘